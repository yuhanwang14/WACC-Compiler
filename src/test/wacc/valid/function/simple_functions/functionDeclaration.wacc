# a simple function is declared, but not called

# Output:

# Program:

begin
  int f() is
    return 0
  end
  skip
end
