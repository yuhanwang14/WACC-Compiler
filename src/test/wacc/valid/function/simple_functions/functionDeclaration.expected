 1
 2
 3
 4
 5
 6
 7
 8
 9
10
11
12
13
# a simple function is declared, but not called

# Output:

# Program:

begin
  int f() is
    return 0
  end
  skip
end
 